apps/Makefile

```diff
@@ -37,7 +37,7 @@ $(addsuffix .prg,$(TCP)): IP65LIB = ../ip65/ip65_tcp.lib
 $(addsuffix .bin,$(UDP)): IP65LIB = ../ip65/ip65.lib
 $(addsuffix .bin,$(TCP)): IP65LIB = ../ip65/ip65_tcp.lib
 
-$(addsuffix .com,$(UDP)): IP65LIB = ../ip65/ip65.lib
+$(addsuffix .com,$(UDP)) datex.com: IP65LIB = ../ip65/ip65.lib
```

Comment:
I must admit that really dislike adding a explicit program to all those generic rules. I suggest that you find a way to express the general idea that every .c file is supposed to be compiled for yet another cc65 target.

Reply:
Believe me, I thought quite hard how to add that new file. (But it was also already quite late.)
The Makefile currently has no notion about ".c" files or ".s" files. It only defines "UDP" files and "TCP" files, extension doesn't matter. If you tell me how to do it without rewriting the file in large part, I'm happy to do that.